ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/vlan-internal-order.j2

```diff
@@ -1,5 +1,5 @@
 {# Internal VLAN allocation policy  #}
-{% if vlan_internal_order is defined and vlan_internal_order is not none %}
+{% if vlan_internal_order is arista.avd.defined %}
```

Comment:
change to check for `vlan_internal_order.allocation` *and* `vlan_internal_order.range.beginning` *and* `vlan_internal_order.range.ending`